internal/app/cmd/cmd.go

```diff
@@ -135,6 +141,9 @@ func runServer(cfg killgrave.Config) server.Server {
 		proxyServer,
 		cfg.Secure,
 		imposterFs,
+		server.PrepareAccessControl(cfg.CORS),
```

Comment:
Same as above, I don't think this refactor is related (and/or strictly required) with the purpose of this pull request. Plus, I'm not even sure we really want it.

Reply:
This is needed so I can propagate cors here: https://github.com/friendsofgo/killgrave/pull/170/files#diff-09b422118d0773bb206a9e4cf4cbbaeeb017894e4007525bfbeb8d0163b3c824R102-R106